packages/components/order-tracking/src/index.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import translate from 'angular-translate';
+import component from './order-tracking.component';
+
+import './index.scss';
+
+const moduleName = 'orderTracking';
+const componentName = 'orderTrackingComponent';
+
+angular
+  .module(moduleName, [
+    'ngTranslateAsyncLoader',
```

Comment:
Missing import statements for:
- ngTranslateAsyncLoader
- ovh-api-services
- ovh-ui-angular